src/modules/previewpane/StlThumbnailProvider/StlThumbnailProvider.csproj

```diff
@@ -0,0 +1,65 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+  <PropertyGroup>
+    <Platforms>x64</Platforms>
+    <UseWindowsForms>true</UseWindowsForms>
+    <ProjectGuid>{F7C8C0F1-5431-4347-89D0-8E5354F93CF2}</ProjectGuid>
+    <RootNamespace>Microsoft.PowerToys.ThumbnailHandler.Stl</RootNamespace>
+    <AssemblyName>PowerToys.StlThumbnailProvider</AssemblyName>
+    <AssemblyTitle>PowerToys.StlThumbnailProvider</AssemblyTitle>
+    <AssemblyDescription>PowerToys StlPreviewHandler</AssemblyDescription>
+    <AssemblyCompany>Microsoft Corporation</AssemblyCompany>
+    <AssemblyCopyright>Copyright (C) 2020 Microsoft Corporation</AssemblyCopyright>
+    <AssemblyProduct>PowerToys</AssemblyProduct>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
we just switched to .NET 5. Please update accordingly

Reply:
Done!